Dockerfile

```diff
@@ -164,10 +165,13 @@ RUN apk add --no-cache \
         bash~=5 \
         openssh~=9 \
         dumb-init~=1 \
-        gcompat~=1
+        gcompat~=1 \
+        su-exec~=0
```

Comment:
I'm not familiar with Alpine, but is there a reason for using  `su-exec` instead of [`gosu` proper?](https://pkgs.alpinelinux.org/packages?name=gosu&branch=edge&repo=&arch=&maintainer=)

Reply:
The gosu readme suggested su-exec specifically for alpine users since they do the same thing. See [Install.md](https://github.com/tianon/gosu/blob/master/INSTALL.md) from gosu repo.

Reply:
Ah, I wonder if its recommended purely from size POV or technical reasons - if its pure size, I would probably prefer the native `gosu` to keep the number of moving parts (and attack/bug vectors) low. 

I know alpine users _generally_ are obsessed with shaving bytes off images, but in this case the trade-off doesn't seem favourable in that regard to me.

If there are stability/technical reasons for it, thats of course a different story entirely :) 

(again, not a maintainer, just an active user of Atlantis voicing an opinion)